src/Models/SceneConnection.php

```diff
@@ -15,15 +15,15 @@ class SceneConnection
 {
     /**
      * @Id
-     * @ManyToOne(targetEntity="Scene", inversedBy="outgoingConnections", cascade={"persist"})
-     * @JoinColumn(name="outgoing_scene_id", referencedColumnName="id")
+     * @ManyToOne(targetEntity="Scene")
+     * @JoinColumn(name="outgoingScene", referencedColumnName="id")
```

Comment:
nit: I thought our convention for table names/column names used underscore_case instead of camelCase? I dont really care just want to be consistent.